ui/src/containers/NavigationModalContainer.js

```diff
@@ -20,7 +19,8 @@ class NavigationModalContainer extends React.Component {
 
   render () {
     return (
-      <LargeModal
+      <Modal
+        isLarge
         title={this.props.title}
         isOpen={this.props.isOpen}
         onClose={this.handleModalClose}
```

Comment:
Add `onClose` to required props?

Reply:
I don't believe the onClose should be passed in to any of the modals. The modals themselves should be taking care of calling the redux action. I will include this in a larger refactor PR this week.